test/shared/cleanup/languages/en.js

```diff
@@ -0,0 +1,35 @@
+import test from 'ava';
+import * as en from '../../../../shared/cleanup/languages/en';
+
+function clean(t, input, expected) {
+  const cleaned = en.clean(input);
+	t.deepEqual(cleaned, expected);
+}
+
+test('[EN] removes multiple spaces', clean, ['this is  wrong'], ['this is wrong']);
```

Comment:
I don't know how [EN] is helpful when the test path already has `en` in it, but you do you.